### ISSUE TITLE
[SF][a11y][valueStateMessage] Error state in valueStateMessage is not announced by screen reader

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#5495]
### **Bug Description**
The valueState is not announced with the text in valueStateMessage, while the state is actually announced in sapui5 library.
![image](https://user-images.githubusercontent.com/1867889/178464147-542d7344-5823-488f-bc45-ffa9452aa323.png)

### **Expected Behavior**
The valueState should be also announced to align with the behavior in sapui5 library.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Input/
2. Switch to `Morning Horizon (Light)` theme
3. Go to section `Input with Suggestions and Value State message`
4. Trigger out the value state message in any input control
5. Check if the value state is announced by screen reader or not.

### **Context**
- UI5 Web Components version: 1.5.0
- OS/Platform: Mac OS
- Browser: Edge
- Affected component: ui5-input, ui5-date-picker, ui5-select, ui5-multi-input, ui5-combobox, ui5-multi-combobox

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SF
- Business impact: {...}

**Comment by ndeshev** [comment]
Hello @SAP/ui5-webcomponents-topic-p all components are now fixed except `ui5-select`, could you please take over this issue.
